stage #4/module #5. AWS/theory/dynamodb.md

```diff
@@ -0,0 +1,143 @@
+# AWS DynamoDB
```

Comment:
@a1exanddrovich Is it possible to add some pictures/diagrams for more visibility?

Reply:
FIxed